### PR TITLE
Updated pkg version on bump

### PR DESCRIPTION
**Pull request by dsimog01**
Package version on manifest is now updated on each bump for upstream versions

### CODE REVIEW

File: src/commands/githubActions/bumpUpstream/index.ts [hunk 1]
```diff
@@ -153,6 +157,24 @@ Compose - ${JSON.stringify(compose, null, 2)}
 
   const versionsToUpdate = Array.from(versionsToUpdateMap.values());
   manifest.upstreamVersion = getUpstreamVersionTag(versionsToUpdate);
+
+  const ethProvider = await tryEthProviders({
```
Comment (pablomendezroyo) [inline src/commands/githubActions/bumpUpstream/index.ts:161]: What if you provide your own provider? it should be contemplated as the first opt

File: src/utils/tryEthProviders.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+ * Logs error messages and fallback attempts for each failed provider.
+ * @param {string[]} providers - An array of Ethereum provider names (e.g., ['dappnode', 'infura']).
+ * @returns {Promise<string>} A promise that resolves to the first successfully connected provider's name
+ * or an empty string if none of the providers could be connected.
+ */
+export async function tryEthProviders({
+  providers
+}: {
+  providers: string[];
+}): Promise<string> {
```
Comment (pablomendezroyo) [inline src/utils/tryEthProviders.ts:14]: I would return srting | undefined. If no provider then return undefined. Handle this case afterwards

File: src/commands/githubActions/bumpUpstream/index.ts [hunk 3]
```diff
@@ -153,6 +157,24 @@ Compose - ${JSON.stringify(compose, null, 2)}
...
+    providers: ["dappnode", "infura"]
```
Comment (pablomendezroyo) [inline src/commands/githubActions/bumpUpstream/index.ts:162]: Arent these providers somewhere in params already?

**Review by pablomendezroyo — CHANGES_REQUESTED** [review]

File: src/utils/tryEthProviders.ts (hunk 2 above)
Comment (dsimog01) [reply src/utils/tryEthProviders.ts]: Done!

File: src/commands/githubActions/bumpUpstream/index.ts (hunk 3 above)
Comment (dsimog01) [reply src/commands/githubActions/bumpUpstream/index.ts:162]: In order places of the code the provider is being passed as a string like here

File: src/commands/githubActions/bumpUpstream/index.ts (hunk 1 above)
Comment (dsimog01) [reply src/commands/githubActions/bumpUpstream/index.ts:161]: Where would you provide it? The `CliGlobalOptions` the gaBumpUpstreamHandler` is receiving does not allow this parameter

Comment (pablomendezroyo) [reply src/commands/githubActions/bumpUpstream/index.ts:161]: Consider adding it

**Review by pablomendezroyo — CHANGES_REQUESTED** [review]

File: src/utils/tryEthProviders.ts [hunk 4]
```diff
@@ -0,0 +1,25 @@
+import { verifyEthConnection } from "./verifyEthConnection.js";
+
+/**
+ * Tries to connect to Ethereum providers in the order they appear in the input array.
+ * Logs error messages and fallback attempts for each failed provider.
+ * @param {string[]} providers - An array of Ethereum provider names (e.g., ['dappnode', 'infura']).
+ * @returns {Promise<string>} A promise that resolves to the first successfully connected provider's name
+ * or an empty string if none of the providers could be connected.
+ */
+export async function tryEthProviders({
```
Comment (pablomendezroyo) [inline src/utils/tryEthProviders.ts:10]: This function should be renamed to something like `getEthProvider`

**Review by pablomendezroyo — CHANGES_REQUESTED** [review]

**Review by pablomendezroyo — APPROVED** [review]

**Pull request merged**